CONTRIBUTING.md

```diff
@@ -80,27 +70,39 @@ Must be one of the following:
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, etc)
 * **refactor**: A code change that neither fixes a bug nor adds a feature
 * **perf**: A code change that improves performance
+* **revert**: Reverts a previous commit
 * **test**: Adding missing tests
 * **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
   generation
 
-### Scope
+##### Revert
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `Revert commit <hash>.`, where the hash is the SHA of the commit being reverted.
+
+#### Scope
 The scope could be anything specifying place of the commit change. For example `$location`,
-`$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc...
+`$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc.
 
-### Subject
+#### Subject
 The subject contains succinct description of the change:
 
 * use the imperative, present tense: "change" not "changed" nor "changes"
 * don't capitalize first letter
 * no dot (.) at the end
 
+A properly formed git commit subject line should always be able to complete the following sentence:
+
+> If applied, this commit will ___your subject line here___
```

Comment:
Yes, I copied the text from [here](http://chris.beams.io/posts/git-commit/) and adapted the format. In that context, a quote looks ad hoc for me. Like quoting the committer's brain.